docs/reference/cli.md

```diff
@@ -214,3 +214,63 @@ if-env --install
 ```sh
 if-env --manifest executed-manifest-path.yml --install
 ```
+
+
+## `if-env`
```

Comment:
@jmcook1186 I've added the description of the `if-env` in line 200. Could you please update it

Reply:
haha oops.